src/constants/v3/tokens.ts

```diff
@@ -0,0 +1,120 @@
+import { Token } from '@uniswap/sdk-core';
+import { SupportedChainId } from './chains';
+import { Matic } from 'v3lib/entities/matic';
+import { WMATIC } from 'v3lib/entities/wmatic';
+
+// export const USDC_BINANCE = new Token(
+//   SupportedChainId.BINANCE,
+//   '0x37792237f932004b9f07BB55C3a3ad77e3BF4ca1',
+//   8,
+//   'USDC',
+//   'USD//C'
+// )
+
+export const WETH_POLYGON = new Token(
```

Comment:
No need for this file it's a duplicate of globalvalue consts.

See what I did in my branch I just made a method to convert old token to new called toToken

Reply:
Okay, fixing this now